dandischema/datacite.py

```diff
@@ -176,21 +176,39 @@ def to_datacite(
         attributes["relatedIdentifiers"] = []
         for rel_el in meta.relatedResource:
             if rel_el.identifier is not None:
-                if ":" not in rel_el.identifier:
-                    continue
-                ident_tp, ident_id = rel_el.identifier.split(":", 1)
-                if ident_tp.lower() in DATACITE_MAP:
-                    ident_tp = DATACITE_MAP[ident_tp.lower()]
-                else:
-                    raise ValueError(
-                        f"identifier has to be from the list: {DATACITE_IDENTYPE}, "
-                        f"but {ident_tp} provided"
-                    )
+                if "doi:" in rel_el.identifier.lower():
```

Comment:
Why `in` and not `.startswith`? If some free text with embedded doi, url - might error